sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ModuleIdentityLifecycleSamples.cs

```diff
@@ -75,7 +74,7 @@ public async Task<DeviceIdentity> CreateDeviceIdentityAsync(string deviceId)
                 // Call APIs to create the device identity.
                 Response<DeviceIdentity> response = await IoTHubServiceClient.Devices.CreateOrUpdateIdentityAsync(deviceIdentity);
```

Comment:
+1 for the sub-client here `IoTHubServiceClient.Devices.CreateOrUpdateIdentityAsync()`

Reply:
yay